test/AWS.Deploy.CLI.IntegrationTests/Helpers/Utilities.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using AWS.Deploy.Orchestration.Utilities;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace AWS.Deploy.CLI.IntegrationTests.Helpers
+{
+    public static class Utilities
+    {
+        /// <summary>
+        /// This method sets a custom workspace which will be used by the deploy tool to create and run the CDK project and any temporary files during the deployment.
+        /// It also adds a nuget.config file that references a private nuget-cache. This cache holds the latest (in-development/unreleased) version of AWS.Deploy.Recipes.CDK.Common.nupkg file
+        /// </summary>
+        public static void OverrideDefaultWorkspace(ServiceProvider serviceProvider, string customWorkspace)
+        {
+            var environmentVariableManager = serviceProvider.GetRequiredService<IEnvironmentVariableManager>();
+            environmentVariableManager.SetEnvironmentVariable("AWS_DOTNET_DEPLOYTOOL_WORKSPACE", customWorkspace);
+            Directory.CreateDirectory(customWorkspace);
+
+            var nugetCachePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".aws-dotnet-deploy", "Projects", "nuget-cache");
+            nugetCachePath = nugetCachePath.Replace(Path.DirectorySeparatorChar, '/');
```

Comment:
Why are we replacing an OS-agnostic separator with an OS-specific one? Does this test work on both Windows and Linux?

Reply:
The '/' should work on windows, but if we use '\' in windows, then we will need to escape that character in json. GIven that we ran the test on Windows and it worked, if the the PR check passes on linux, we should be good to go.

Reply:
Okay, fine if both Windows and Linux passed.